packages/node/src/models/proposed-app-instance-info.ts

```diff
@@ -128,13 +121,15 @@ export class ProposedAppInstanceInfo implements AppInstanceInfo {
       signingKeys,
       bigNumberify(this.timeout).toNumber(),
       proposedAppInterface,
-      proposedTerms,
       isVirtualApp,
       isVirtualApp ? 1337 : stateChannel.numInstalledApps,
       stateChannel.rootNonceValue,
       this.initialState,
       0,
-      bigNumberify(this.timeout).toNumber()
+      bigNumberify(this.timeout).toNumber(),
+      // not used in app identity computation
```

Comment:
Is this comment trying to say "the below 2 parameters are not used in app identity computation"? It is unclear.